### ISSUE TITLE
CT error: Assertion failed: reentry detected

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/friction#97]
friction : accessibility-fuzz : require.js : run
Uncaught Error: Assertion failed: reentry detected, value=1, oldValue=11.829334000000001
Error: Assertion failed: reentry detected, value=1, oldValue=11.829334000000001
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/assert/js/assert.js:22:13)
    at NumberProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:301:19)
    at NumberProperty.setValueAndNotifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:287:14)
    at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:230:16)
    at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/NumberProperty.js?bust=1535068429421:86:30)
    at NumberProperty.reset (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:327:14)
    at FrictionModel.reset (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/friction/js/friction/model/FrictionModel.js?bust=1535068429421:339:30)
    at Array.listener (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/friction/js/friction/view/FrictionScreenView.js?bust=1535068429421:171:15)
    at Emitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Emitter.js?bust=1535068429421:134:49)
    at PushButtonModel.fire (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/sun/js/buttons/PushButtonModel.js?bust=1535068429421:155:25)
Approximately 8/23/2018, 5:33:16 PM

**Comment by jbphet** [comment]
I spent a few minutes looking at this, and it is the `amplitudeProperty` of `FrictionModel` that seems to be experiencing reentry based on the stack trace shown above.  I was unable to duplicate the problem after 20 minutes of fuzz testing.  I was using a slightly different version of code because I'd commented out some code for alerts that was causing a different assertion during CT, see #96.  As a test, I added back the common code, and was able to duplicate the stack trace above by experiencing the first problem, then hitting the 'continue' button.  So, I think the problem in #96 is also causing this issue.

Assigning to @zepumph, and I'd suggest fixing the other CT issue first and then seeing if it also causes this one to go away.  Based on what I'm seeing, I believe it will.
